keps/sig-api-machinery/4020-unknown-version-interoperability-proxy/README.md

```diff
@@ -397,6 +376,7 @@ We will test the feature mostly in integration test and unit test. We may add e2
 
 #### Beta
 
+- Version aware proying implemented
```

Comment:
s/proying/proxying

Reply:
Actually removed the Version aware proxying bit from this PR since I modified this PR to only focus on "replacing SV API with Agg Discovery"